### PR TITLE
Add fixes after live testing with more services

### PR DESCRIPTION
**Pull request by sbtsrbayer**
From the documentation:
https://apereo.github.io/cas/5.2.x/protocol/CAS-Protocol-V2-Specification.html
"If the certificate fails validation, no proxy-granting ticket will be issued, and the CAS service response as described in Section 2.5.2 MUST NOT contain a block. "

I had to remove this from the xml response view to get Canvas to authenticate.

Canvas also defaulted to the proxyService eventhough it didn't need to proxy.

There were some routes expected by 3rd parties that needed to be added too.

### CODE REVIEW

**Review by loren138 — APPROVED** [review]

File: resources/routes.php [hunk 1]
```diff
@@ -16,9 +16,15 @@
...
+Route::get('/p3/logout', '\Loren138\CASServer\Http\Controllers\CasController@getLogout');
+
 // CAS 1.0 Validate
 Route::get('/validate', '\Loren138\CASServer\Http\Controllers\CasController@getValidate');
 // CAS 2.0 Validate
 Route::get('/serviceValidate', '\Loren138\CASServer\Http\Controllers\CasController@getServiceValidate');
+Route::get('/proxyValidate', '\Loren138\CASServer\Http\Controllers\CasController@getProxyValidate');
 // CAS 3.0 Validate
 Route::get('/p3/serviceValidate', '\Loren138\CASServer\Http\Controllers\CasController@getServiceValidate3');
+Route::get('/p3/proxyValidate', '\Loren138\CASServer\Http\Controllers\CasController@getProxyValidate3');
```
Comment (loren138) [inline resources/routes.php:30]: Can you add tests for the 5 new endpoints (even if some of them are duplicates of existing tests)?  (You could just factor out the duplicate code into a test helper method.)

**Review by loren138 — CHANGES_REQUESTED** [review]
